target/linux/ipq40xx/files-6.12/arch/arm/boot/dts/qcom/qcom-ipq4029-brulk-common.dtsi

```diff
@@ -0,0 +1,579 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/* Copyright (c) 2025, Natalie Moore <natalie@natalielucy.dev> */
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/thermal/thermal.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+	/* Common device tree fragments for both IBR1700 (Hulk) and AER2200 (Bruce) */
+	aliases {
+		// TODO: Verify if the ethernet0 alias is needed
+		ethernet0 = &gmac;
```

Comment:
please also add aliases for leds (led-boot, led-failsafe, led-running, led-upgrade) and label-mac-device.

Reply:
The SS leds are signal strength - visual indication of 1-4 "bars" of cell reception.

I think defining just upgrade and failsafe LEDs would work well enough in this case. Do you agree?

Reply:
yep, better then nothing. booting indication would be nice but i'm out of ideas in your case. maybe someone else has a better solution for these devices.

Reply:
I'll push the change and leave this comment open if another reviewer has any opinions.

Edit: To add a bit, in vendor firmware the signal LEDs do a little "knight rider"-style animation until the router is up. I think that might be a bit too complex/specific to implement at a wider level, but if there's a good hook to add it in for the device specifically I'm down for that.